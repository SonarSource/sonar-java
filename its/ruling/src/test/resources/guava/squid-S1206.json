{
'com.google.guava:guava:src/com/google/common/base/Converter.java':[
382,
],
'com.google.guava:guava:src/com/google/common/collect/AbstractListMultimap.java':[
120,
],
'com.google.guava:guava:src/com/google/common/collect/AbstractSetMultimap.java':[
131,
],
'com.google.guava:guava:src/com/google/common/collect/Cut.java':[
89,
286,
351,
],
'com.google.guava:guava:src/com/google/common/collect/EmptyImmutableSortedMultiset.java':[
92,
],
'com.google.guava:guava:src/com/google/common/collect/HashBasedTable.java':[
138,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableMultiset.java':[
401,
],
'com.google.guava:guava:src/com/google/common/collect/Multimaps.java':[
1056,
],
'com.google.guava:guava:src/com/google/common/collect/RegularImmutableBiMap.java':[
226,
294,
],
'com.google.guava:guava:src/com/google/common/collect/RegularImmutableMultiset.java':[
76,
],
'com.google.guava:guava:src/com/google/common/collect/RegularImmutableSet.java':[
87,
],
'com.google.guava:guava:src/com/google/common/collect/RegularImmutableSortedSet.java':[
149,
],
'com.google.guava:guava:src/com/google/common/collect/StandardTable.java':[
369,
],
'com.google.guava:guava:src/com/google/common/collect/Synchronized.java':[
890,
],
}
