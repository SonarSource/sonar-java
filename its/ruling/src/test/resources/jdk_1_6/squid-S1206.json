{
'jdk_1_6:java/awt/AttributeValue.java':[
28,
],
'jdk_1_6:java/awt/Point.java':[
195,
],
'jdk_1_6:java/awt/Rectangle.java':[
1177,
],
'jdk_1_6:java/awt/font/TextLine.java':[
852,
],
'jdk_1_6:java/awt/image/BandedSampleModel.java':[
818,
],
'jdk_1_6:java/awt/image/ComponentColorModel.java':[
2919,
],
'jdk_1_6:java/awt/image/PackedColorModel.java':[
374,
],
'jdk_1_6:java/awt/image/PixelInterleavedSampleModel.java':[
150,
],
'jdk_1_6:java/sql/Timestamp.java':[
427,
],
'jdk_1_6:java/util/Collections.java':[
1403,
],
'jdk_1_6:java/util/EnumMap.java':[
616,
],
'jdk_1_6:java/util/IdentityHashMap.java':[
971,
],
'jdk_1_6:java/util/JumboEnumSet.java':[
324,
],
'jdk_1_6:java/util/RegularEnumSet.java':[
275,
],
'jdk_1_6:java/util/concurrent/ConcurrentSkipListMap.java':[
1855,
2346,
2456,
],
'jdk_1_6:java/util/concurrent/ConcurrentSkipListSet.java':[
263,
],
'jdk_1_6:java/util/concurrent/CopyOnWriteArraySet.java':[
327,
],
'jdk_1_6:java/util/jar/JarVerifier.java':[
581,
],
'jdk_1_6:java/util/zip/ZipEntry.java':[
316,
],
}
