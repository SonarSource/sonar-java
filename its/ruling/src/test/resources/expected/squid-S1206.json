{
'project:com/sun/corba/se/impl/javax/rmi/CORBA/StubDelegateImpl.java':[
134,
],
'project:com/sun/corba/se/impl/orb/ParserTable.java':[
451,
514,
539,
568,
604,
861,
878,
947,
978,
],
'project:com/sun/corba/se/impl/orbutil/RepIdDelegator.java':[
171,
],
'project:com/sun/corba/se/impl/orbutil/RepIdDelegator_1_3.java':[
171,
],
'project:com/sun/corba/se/impl/orbutil/RepIdDelegator_1_3_1.java':[
171,
],
'project:com/sun/corba/se/impl/protocol/CorbaClientDelegateImpl.java':[
437,
],
'project:com/sun/corba/se/impl/transport/CorbaContactInfoListImpl.java':[
138,
],
'project:com/sun/jmx/snmp/SnmpOidTableSupport.java':[
143,
],
'project:com/sun/jmx/snmp/SnmpParameters.java':[
144,
],
'project:com/sun/org/apache/bcel/internal/generic/BasicType.java':[
100,
],
'project:com/sun/org/apache/bcel/internal/generic/Instruction.java':[
304,
],
'project:com/sun/org/apache/bcel/internal/generic/LocalVariableGen.java':[
187,
],
'project:com/sun/org/apache/bcel/internal/generic/ReturnaddressType.java':[
92,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/FunctionCall.java':[
158,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/VariableRefBase.java':[
100,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/util/NodeType.java':[
71,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/AbstractDateTimeDV.java':[
953,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/DecimalDV.java':[
219,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/PrecisionDecimalDV.java':[
148,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/QNameDV.java':[
93,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/util/ShortListImpl.java':[
92,
],
'project:com/sun/org/apache/xerces/internal/jaxp/datatype/DurationImpl.java':[
976,
],
'project:com/sun/org/apache/xerces/internal/util/URI.java':[
1686,
],
'project:com/sun/org/apache/xerces/internal/util/XMLEntityDescriptionImpl.java':[
144,
],
'project:com/sun/org/apache/xerces/internal/util/XMLResourceIdentifierImpl.java':[
185,
],
'project:com/sun/org/apache/xerces/internal/xinclude/XIncludeHandler.java':[
2624,
2670,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMNodeProxy.java':[
144,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/ExtendedType.java':[
98,
],
'project:com/sun/org/apache/xml/internal/security/keys/content/x509/XMLX509Certificate.java':[
143,
],
'project:com/sun/org/apache/xml/internal/security/keys/content/x509/XMLX509IssuerSerial.java':[
150,
],
'project:com/sun/org/apache/xml/internal/security/keys/content/x509/XMLX509SKI.java':[
154,
],
'project:com/sun/org/apache/xml/internal/security/keys/content/x509/XMLX509SubjectName.java':[
86,
],
'project:com/sun/org/apache/xml/internal/serializer/utils/URI.java':[
1324,
],
'project:com/sun/org/apache/xml/internal/utils/URI.java':[
1344,
],
'project:com/sun/org/apache/xpath/internal/Arg.java':[
234,
],
'project:com/sun/security/auth/NTSidDomainPrincipal.java':[
96,
],
'project:com/sun/security/auth/NTSidGroupPrincipal.java':[
91,
],
'project:com/sun/security/auth/NTSidPrimaryGroupPrincipal.java':[
94,
],
'project:com/sun/security/auth/NTSidUserPrincipal.java':[
90,
],
'project:java/awt/AttributeValue.java':[
50,
],
'project:java/awt/Point.java':[
214,
],
'project:java/awt/Rectangle.java':[
1196,
],
'project:java/awt/font/TextLine.java':[
881,
],
'project:java/awt/image/BandedSampleModel.java':[
846,
],
'project:java/awt/image/ComponentColorModel.java':[
2936,
],
'project:java/awt/image/PackedColorModel.java':[
396,
],
'project:java/awt/image/PixelInterleavedSampleModel.java':[
168,
],
'project:java/util/Collections.java':[
1461,
],
'project:java/util/IdentityHashMap.java':[
999,
],
'project:java/util/JumboEnumSet.java':[
347,
],
'project:java/util/RegularEnumSet.java':[
294,
],
'project:java/util/concurrent/ConcurrentSkipListMap.java':[
1843,
2335,
2444,
],
'project:java/util/concurrent/ConcurrentSkipListSet.java':[
293,
],
'project:java/util/concurrent/CopyOnWriteArraySet.java':[
354,
],
'project:java/util/jar/JarVerifier.java':[
603,
],
'project:java/util/zip/ZipEntry.java':[
310,
],
'project:javax/management/NotificationBroadcasterSupport.java':[
291,
308,
],
'project:javax/print/attribute/EnumSyntax.java':[
150,
],
'project:javax/print/attribute/standard/Copies.java':[
107,
],
'project:javax/print/attribute/standard/CopiesSupported.java':[
110,
],
'project:javax/print/attribute/standard/DateTimeAtCompleted.java':[
91,
],
'project:javax/print/attribute/standard/DateTimeAtCreation.java':[
91,
],
'project:javax/print/attribute/standard/DateTimeAtProcessing.java':[
92,
],
'project:javax/print/attribute/standard/Destination.java':[
93,
],
'project:javax/print/attribute/standard/DocumentName.java':[
92,
],
'project:javax/print/attribute/standard/JobHoldUntil.java':[
123,
],
'project:javax/print/attribute/standard/JobImpressions.java':[
114,
],
'project:javax/print/attribute/standard/JobImpressionsCompleted.java':[
96,
],
'project:javax/print/attribute/standard/JobImpressionsSupported.java':[
98,
],
'project:javax/print/attribute/standard/JobKOctets.java':[
165,
],
'project:javax/print/attribute/standard/JobKOctetsProcessed.java':[
108,
],
'project:javax/print/attribute/standard/JobKOctetsSupported.java':[
96,
],
'project:javax/print/attribute/standard/JobMediaSheets.java':[
107,
],
'project:javax/print/attribute/standard/JobMediaSheetsCompleted.java':[
97,
],
'project:javax/print/attribute/standard/JobMediaSheetsSupported.java':[
96,
],
'project:javax/print/attribute/standard/JobMessageFromOperator.java':[
97,
],
'project:javax/print/attribute/standard/JobName.java':[
96,
],
'project:javax/print/attribute/standard/JobOriginatingUserName.java':[
97,
],
'project:javax/print/attribute/standard/JobPriority.java':[
98,
],
'project:javax/print/attribute/standard/JobPrioritySupported.java':[
90,
],
'project:javax/print/attribute/standard/Media.java':[
90,
],
'project:javax/print/attribute/standard/MediaSize.java':[
256,
],
'project:javax/print/attribute/standard/NumberOfDocuments.java':[
82,
],
'project:javax/print/attribute/standard/NumberOfInterveningJobs.java':[
81,
],
'project:javax/print/attribute/standard/NumberUp.java':[
163,
],
'project:javax/print/attribute/standard/NumberUpSupported.java':[
147,
],
'project:javax/print/attribute/standard/OutputDeviceAssigned.java':[
96,
],
'project:javax/print/attribute/standard/PageRanges.java':[
236,
],
'project:javax/print/attribute/standard/PagesPerMinute.java':[
83,
],
'project:javax/print/attribute/standard/PagesPerMinuteColor.java':[
94,
],
'project:javax/print/attribute/standard/PrinterInfo.java':[
92,
],
'project:javax/print/attribute/standard/PrinterLocation.java':[
88,
],
'project:javax/print/attribute/standard/PrinterMakeAndModel.java':[
86,
],
'project:javax/print/attribute/standard/PrinterMessageFromOperator.java':[
102,
],
'project:javax/print/attribute/standard/PrinterMoreInfo.java':[
92,
],
'project:javax/print/attribute/standard/PrinterMoreInfoManufacturer.java':[
95,
],
'project:javax/print/attribute/standard/PrinterName.java':[
90,
],
'project:javax/print/attribute/standard/PrinterResolution.java':[
120,
],
'project:javax/print/attribute/standard/PrinterURI.java':[
85,
],
'project:javax/print/attribute/standard/QueuedJobCount.java':[
80,
],
'project:javax/print/attribute/standard/RequestingUserName.java':[
98,
],
'project:javax/swing/text/DefaultCaret.java':[
1429,
],
}
